contrib/py/admin/dumpstats.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python3
```

Comment:
it would be nice to run this through something like [`black`](https://black.readthedocs.io/en/stable/) so we have it nicely formatted.